clients/admin-ui/src/features/config-wizard/DescribeSystemsForm.tsx

```diff
@@ -59,7 +59,8 @@ const DescribeSystemsForm = ({
         },
       ],
       system_type: values.system_type,
-      meta: { tags: values.tags?.toString() },
+      // QUESTION(allisonking): should this be using the system tags now in fideslang 1.1.0?
+      meta: { tags: values.tags?.toString() ?? "" },
```

Comment:
I tried to update this to use the new `tags` field but got an internal server error that I couldn't figure out how to get more info on :(

Reply:
I wonder if the database maybe still needs to be reset for the new `tags` fields? Some of those error messages can certainly be frustrating

Reply:
Hmm, ya that should have taken care of that scenario I would think 🤔 will take a closer look 👀

Reply:
okay it works today, probably just a mistake on my end! 😅